test/server/lib/tax-forms.test.js

```diff
@@ -140,6 +143,13 @@ describe('server/lib/tax-forms', () => {
       year: year - 1,
     });
 
+    // Create legal document for accountWithTaxFormSubmittedByHost (no tax form should be required in this case)
+    await fakeLegalDocument({
+      CollectiveId: accountWithTaxFormSubmittedByHost.HostCollectiveId,
+      requestStatus: 'RECEIVED',
+      year: year,
+    });
+
```

Comment:
The goal of this chunk + the `Expense.create` below is to make sure `accountWithTaxFormSubmittedByHost` is not included in the test `findAccountsThatNeedToBeSentTaxForm` > `returns the right profiles`